test/Typewriter.Test/TestDataPHP/com/Microsoft/Graph/Model/Entity.php

```diff
@@ -118,7 +118,7 @@ public function jsonSerialize()
     {
         $serializableProperties = $this->getProperties();
         foreach ($serializableProperties as $property => $val) {
-            if (is_a($val, "\DateTime")) {
+            if (get_class((object)$val) === "\DateTime")) {
```

Comment:
Guessing we should expect changes to some setters and getters as well reflecting the new types?